app.py

```diff
@@ -33,6 +35,97 @@ def add_new_user(email, user_id, user_name, auth_type, profile_picture):
             db.session.commit()
 
 
+def get_new_citation(url):
+    if re.match(regex, url) is not None:
+        # response=python_requests.get(url)
+        # ---Sample of a successful response with null values
+        responseNull = """{
```

Comment:
We can use this for mocked responses from the Microlink API for unit testing, but we should keep them out of the main app.